src/client/pages/OfferNew/InsuranceSelector/index.tsx

```diff
@@ -45,9 +56,25 @@ export const InsuranceSelector: React.FC<Props> = ({
   })
 
   return (
-    <Selector
-      insurances={insurances}
-      onChange={(id: string) => onChange(variantMap[id])}
-    />
+    <ContainerWrapper>
+      <Container>
+        <Column>
+          <HeadingWrapper>
+            <HeadingBlack>
+              {textKeys.OFFER_BUNDLE_SELECTOR_TITLE()}
+            </HeadingBlack>
+            <Body>
+              {selectedQuoteBundle &&
+                textKeys.OFFER_BUNDLE_SELECTOR_DESCRIPTION()}
```

Comment:
Why do we need to hide the description text? This only happens when we are loading data, right?

Reply:
You're right, this is left from the time we had the date as a parameter to the textKey.